core/bin/dust_api.rs

```diff
@@ -143,6 +143,95 @@ async fn projects_create(
     }
 }
 
+/// Clones a project.
+/// Simply consists in cloning the latest dataset versions, as we don't copy runs and hence specs.
+
+async fn projects_clone(
```

Comment:
I think it might make more sense to call this function `projects_fork` and the endpoint `/projects/<projectId>/fork`. To me it's a clearer description of what's going on here and consistent with the UI.